bats/tests/k8s/wasm.bats

```diff
@@ -25,7 +25,7 @@ assert_traefik_crd_established() {
 }
 
 @test 'verify no runtimeclasses have been defined' {
-    run kubectl get runtimeclasses --output json
+    run --separate-stderr kubectl get runtimeclasses --output json
```

Comment:
Should we check here, that if `$stderr` is a deprecation warning, and still fail otherwise?

```bash
if [[ -n $stderr ]]; then
    output=$stderr assert_output --partial deprecated
fi
```